x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field_helpers.tsx

```diff
@@ -382,3 +383,16 @@ export const renderUrl = ({
     getEmptyTagValue()
   );
 };
+
+interface RenderRiskScoreProps {
+  value: number | string;
+}
+
+export const RenderRiskScore: React.FC<RenderRiskScoreProps> = ({ value }) => {
```

Comment:
This component is only used once. So, it makes more sense to call `formatRiskScore` directly on x-`formatted_field.tsx`. If you want to keep the new component, I would argue that it should be a reusable component inside the entity analytics folder and be used everywhere we render risk score.

Reply:
Gotcha! I removed the component and called it directly, since it is a simple wrapper, I think we can keep it to the formatter as you suggested :) Thanks!